packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -12926,6 +12953,14 @@ module ZIndexes {
 // WARNING: Unsupported export: DefaultProps
 // WARNING: Unsupported export: Spinner
 // WARNING: Unsupported export: SpinnerLabelPosition
+// WARNING: Unsupported export: StackItem

```

Comment:
@natalieethell or @aditima What ever happened to the work being done on `api-extractor` to support basic `const` exports?

Reply:
This is supported in api-extractor 7